### PR TITLE
home page template

### PR DESCRIPTION
**Pull request by codyjamesbrooks**
Reason for Change
=================
Create a template for the website home page. Launching an event websites home page will be as easy as filling in the content block of the template. 

Changes
=======
Add, and registers a customer website home page template

### CODE REVIEW

File: app/views/staff/pages/themes/default/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  </div>
+</div>
+
+<!-- Opportunities Component -->
+<h4 class="text-center text-3xl font-bold tracking-wider mb-6">Opportunities</h4>
+<div class="mb-6 flex flex-col lg:flex-row items-center lg:justify-center gap-4 max-w-5xl mx-auto mb-12">
+  <div class="w-full lg:w-2/5 flex flex-row lg:flex-col">
+  <%= image_tag 'themes/default/image-placeholder.png', class: "w-1/3 lg:w-full mr-3 block mb-4" %>
+    <div>
+      <h6 class="text-xl font-bold mb-3">Speak at RubyConf</h6>
```
Comment (jonsgreen) [inline app/views/staff/pages/themes/default/home.html.erb:67]: @cody it seems like these elements about speaking and CFPs could be made dynamic?

File: app/views/staff/websites/_form.html.haml [hunk 2]
```diff
@@ -9,7 +9,7 @@
       - if website.background.attached?
         = f.label :background, "Current Background", class: "control-label"
         %div
-          = image_tag(website.background)
+          = image_tag(website.background, width: "200px")
```
Comment (jonsgreen) [inline app/views/staff/websites/_form.html.haml:12]: On my latest branch I did a similar thing using resize. By the way, it is good to know about these resize helpers if you don't already know about them: https://bloggie.io/@kinopyo/upgrade-guide-active-storage-in-rails-6

**Review by jonsgreen — APPROVED** [review]
I am approving this @codyjamesbrooks though you may want to make some of the changes I suggested as comments.

File: app/views/staff/pages/themes/default/home.html.erb (hunk 1 above)
Comment (codyjamesbrooks) [reply app/views/staff/pages/themes/default/home.html.erb:67]: 👍

File: app/views/staff/pages/themes/default/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    <%= image_tag 'themes/default/image-placeholder.png' %>
+    <h5 class="mb-2 font-bold text-xl">Become a Scholar</h5>
+    <p class="mb-3">The Opportunity Scholarship is a chance to jump in to learn more about Ruby specifically and software development in general. Pair with a Guide, an experienced programmer or conference-goer who has attended RubyConf before and is already familiar with many key players at the conference.</p>
+    <a class="px-4 py-3 font-bold text-sm bg-gray-200 rounded-lg">Apply</a></div>
+</div>
+<h4 class="text-center text-3xl font-bold tracking-wider mb-6">Thank you to our Sponsors</h4>
+<p class="max-w-2xl mx-auto mb-6">Interested in becoming a sponsor? Check out our Sponsorship Prospectus to learn about sponsorship opportunities. and email sponsors@rubycentral.org to get in touch!</p>
+
+<!-- Sponsors footer -->
+<div data-controller="sponsors-footer" data-sponsors-footer-event-slug-value="rubyconf-2022"></div>
```
[the remark this answers is not in the document]
Comment (codyjamesbrooks) [reply app/views/staff/pages/themes/default/home.html.erb:95]: 👍

File: app/views/staff/websites/_form.html.haml (hunk 2 above)
Comment (codyjamesbrooks) [reply app/views/staff/websites/_form.html.haml:12]: Per ping, will leave this as a width resize for now due to resize throwing errors on SVG. We can take another stab at this if we don't like how it is working.

**Pull request merged**